src/Schema.php

```diff
@@ -606,26 +640,27 @@ public function findUniqueIndexes(TableSchemaInterface $table): array
     }
 
     /**
-     * Extracts the data type for the given column.
+     * Get the data type for the given column.
      *
      * @param ColumnSchemaInterface $column The column schema object.
      *
      * @return string The abstract column type.
      */
-    private function extractColumnType(ColumnSchemaInterface $column): string
+    private function getColumnType(ColumnSchemaInterface $column): string
```

Comment:
Changes for changes?

Reply:
Just looks like `getColumnPhpType()`. It is not so important.

Reply:
Without rename please